docs/source/daml/intro/1_Token.rst

```diff
@@ -34,17 +34,17 @@ DAML specifies what transactions are legal on a DAML Ledger. The rules the DAML
 DAML files and modules
 ----------------------
 
-Each ``.daml`` file defines a *DAML Module*. At the top of each DAML file is a pragma informing the compiler of the language version and the module name:
+Each ``.daml`` file defines a *DAML Module* at the top:
 
 .. literalinclude:: daml/daml-intro-1/Token.daml
   :language: daml
-  :lines: 6, 9
+  :lines: 5, 9
 
-Code comments in DAML are introduced with `--`:
+Code comments in DAML are introduced with ``--``:
 
 .. literalinclude:: daml/daml-intro-1/Token.daml
   :language: daml
-  :lines: 4-9
+  :lines: 1-5
```

Comment:
I changed this to show the copyright message to keep the size:
```
-- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
-- SPDX-License-Identifier: Apache-2.0

-- A DAML file defines a module.
module Token where
```

Cf [current version](https://docs.daml.com/daml/intro/1_Token.html)

Reply:
I'd just do 4-5.